web_external/User/InviteUserView.js

```diff
@@ -0,0 +1,50 @@
+isic.views.InviteUserView = isic.View.extend({
+    events: {
+        'submit #isic-user-invite-form': function (event) {
+            event.preventDefault();
+            // this.$('#isic-user-invite-submit').prop('disabled', true);
+
+            girder.restRequest({
+                path: 'user/invite',
+                data: {
+                    login: this.$('#isic-user-invite-new-login').val(),
+                    email: this.$('#isic-user-invite-new-email').val(),
+                    firstName: this.$('#isic-user-invite-new-firstname').val(),
+                    lastName: this.$('#isic-user-invite-new-lastname').val()
+                },
+                type: 'POST',
+                error: null
+            })
+                .done(function (resp) {
+                    console.log('success', resp);
```

Comment:
This should display a confirmation with:
* The user's new obfuscated name
* The URL of the confirmation link
* Anything else that might be useful
We maybe shouldn't render this confirm in a modal, if it's more awkward to copy text out of.